resolwe_bio/processes/reads_processing/alignmentsieve.py

```diff
@@ -0,0 +1,151 @@
+"""Filter BAM files using alignmentSieve from deeptools"""
+import os
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    Process,
+    SchedulingClass,
+    Persistence,
+    Cmd,
+    StringField,
+    IntegerField,
+    FileField,
+    DataField,
+)
+
+
+class AlignmentSieve(Process):
+    """Filter alignments of BAM files according to specified parameters
+
+    Optionally, it can output a BEDPE file. Program is bundled with
```

Comment:
Is BEDPE supported in this process?

Reply:
Nice catch. This is the description of the tool and the `--BED` in our process is not yet exposed.

Reply:
Let's not implement this option just now. I've removed the wording in the description for the time being.